File: Samples/VB.Net/VBDotNet/ISmpViewDlg.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports SalesMatePlusLib

Namespace CSharpSDKTest
	Partial Public Class ISmpViewDlg
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub PrintInDosModebtn_Click(ByVal sender As Object, ByVal e As EventArgs) Handles PrintInDosModebtn.Click
			Dim m_SmpView As ISmpView

			m_SmpView = New SmpViewClass()

			m_SmpView.PrintInDosMode()
		End Sub

		Private Sub PrintReportbtn_Click(ByVal sender As Object, ByVal e As EventArgs) Handles PrintReportbtn.Click
			Dim m_SmpView As ISmpView

			m_SmpView = New SmpViewClass()

			m_SmpView.PrintReport()
		End Sub
	End Class
End Namespace

File: Samples/C#.Net/CSharpSDKTest/ISmpViewDlg.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using SalesMatePlusLib;

namespace CSharpSDKTest
{
    public partial class ISmpViewDlg : Form
    {
        public ISmpViewDlg()
        {
            InitializeComponent();
        }

        private void PrintInDosModebtn_Click(object sender, EventArgs e)
        {
            ISmpView m_SmpView;

            m_SmpView = new SmpViewClass();

            m_SmpView.PrintInDosMode();
        }

        private void PrintReportbtn_Click(object sender, EventArgs e)
        {
            ISmpView m_SmpView;

            m_SmpView = new SmpViewClass();

            m_SmpView.PrintReport();
        }
    }
}